webpack/set-webpack-public-path-plugin/src/SetPublicPathCurrentScriptPlugin.ts

```diff
@@ -91,12 +91,14 @@ export class SetPublicPathCurrentScriptPlugin extends SetPublicPathPluginBase {
       }
 
       if (hasProblematicLibraryType) {
-        const codeGenerationResults: CodeGenerationResults = compilation.codeGenerationResults;
+        const codeGenerationResults: CodeGenerationResults | undefined = compilation.codeGenerationResults;
         let appliesToAnyChunk: boolean = false;
-        for (const chunk of compilation.chunks) {
-          if (appliesToChunk(chunk, codeGenerationResults)) {
-            appliesToAnyChunk = true;
-            break;
+        if (codeGenerationResults) {
```

Comment:
Seems like if this is undefined, this should be an error case. @dmichon-msft/@TheLarkInn - thoughts?

Reply:
https://github.com/webpack/webpack/blob/0c213cecf2906bc41102c3a4cfdd1ad3522d0171/lib/CodeGenerationResults.js#L65-L76 it looks like there are multiple cases in which CodeGenerationResults can be undefined. 

https://github.com/webpack/webpack/blob/0c213cecf2906bc41102c3a4cfdd1ad3522d0171/lib/CodeGenerationResults.js#L52

Reply:
So this looks correct to me